README.md

```diff
@@ -150,18 +178,19 @@ matrix:
   allow_failures:
     - php: 5.5
 
-before_script:
+install:
   - curl -s http://getcomposer.org/installer | php
   - php composer.phar install --dev --no-interaction
-
+  - wget https://github.com/satooshi/php-coveralls/releases/download/v0.7.0/coveralls.phar
+  - chmod +x coveralls.phar
 script:
   - mkdir -p build/logs
   - php vendor/bin/phpunit -c phpunit.xml.dist
 
-after_script:
-  - travis_retry php vendor/bin/coveralls
+after_success:
+  - travis_retry php coveralls.phar
   # or enable logging
-  - travis_retry php vendor/bin/coveralls -v
+  - travis_retry php coveralls.phar -v
```

Comment:
I don't like chaning the Travis example. Most repos are configured to cache Composer, and if not then still Travis has it's own cache. By default phar file won't be cached.